mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetLoanTokensList.tsx

```diff
@@ -117,16 +142,90 @@ export const BottomSheetLoanTokensList = ({
+            <ThemedTextV2
+              style={tailwind("text-xl font-normal-v2 pb-5")}
+              light={tailwind("text-mono-light-v2-900")}
+              dark={tailwind("text-mono-dark-v2-900")}
+            >
+              {translate(
+                "components/BottomSheetLoanTokensList",
+                "Select Token"
+              )}
+            </ThemedTextV2>
+            <SearchInputV2
+              testID="loan_search_input"
+              ref={searchRef}
+              value={searchString}
+              showClearButton={searchString !== ""}
+              placeholder={translate(
+                "components/BottomSheetLoanTokensList",
+                "Search with token name"
+              )}
+              containerStyle={tailwind([
+                "border-0.5",
+                isSearchFocus
+                  ? {
+                      "border-mono-light-v2-800": isLight,
+                      "border-mono-dark-v2-800": !isLight,
+                    }
+                  : {
+                      "border-mono-light-v2-00": isLight,
+                      "border-mono-dark-v2-00": !isLight,
+                    },
+              ])}
+              onClearInput={() => {
+                setSearchString("");
+                searchRef?.current?.focus();
+              }}
+              onChangeText={(text: string) => {
+                setSearchString(text);
+              }}
+              onFocus={() => {
+                setIsSearchFocus(true);
+              }}
+              onBlur={() => {
+                setIsSearchFocus(false);
+              }}
+            />
+            {inSearchMode && (
+              <View style={tailwind("mt-5 mx-5")}>
+                <ThemedTextV2
+                  style={tailwind("text-xs pl-5 font-normal-v2")}
+                  light={tailwind("text-mono-light-v2-700")}
+                  dark={tailwind("text-mono-dark-v2-700")}
+                  testID="empty_search_result_text"
+                >
+                  {searchString.trim() === ""
+                    ? translate(
+                        "components/BottomSheetLoanTokensList",
+                        "Search with token name"
+                      )
+                    : translate(
+                        "components/BottomSheetLoanTokensList",
+                        "Search results for “{{searchTerm}}”",
+                        { searchTerm: searchString }
+                      )}
+                </ThemedTextV2>
+              </View>
+            )}
+          </View>
         }
       />
     );
   });
-};
+
+function filterLoanTokensBySearchTerm(
+  loanTokens: LoanToken[],
+  searchTerm: string,
+  isFocused: boolean
+): LoanToken[] {
+  if (searchTerm === "") {
+    return isFocused ? [] : loanTokens;
+  }
+  return loanTokens.filter((t) => {
+    return [t.token.displaySymbol].some((searchItem) =>
+      searchItem.toLowerCase().includes(searchTerm.trim().toLowerCase())
+    );
```

Comment:
```suggestion
    return t.token.displaySymbol
      .toLowerCase()
      .includes(searchTerm.trim().toLowerCase());
```
can simplify it like this? whacha think

Reply:
Looks better